docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -56,6 +56,33 @@ Different settings can be applied to a load balancer service in AWS using _annot
 
 The information for the annotations for AWS is taken from the comments on [aws.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/aws.go)
 
+# OPENSTACK
+This section describes all the possible configurations which can
+be used while setting Kubernetes as an OpenStack provider.
```

Comment:
how about `when using OpenStack with Kubernetes`

Reply:
agreed. This is not about using Kubernetes as an OpenStack provider, it is about using OpenStack to provision Kubernetes, right?